### DISCUSSION TITLE
分块所依赖的向量模型能使用别的模型吗？比如glm的embedding-3？

### DISCUSSION THREAD

**Discussion by BiFangKNT** [lobehub/lobehub#4062] [Ideas]
如题

**Comment by 1341797533syh-cell** [comment]
同问

**Comment by SpeedupMaster** [comment]
#5177
等这个合并

**Comment by arvinxx** [comment]
已经支持了

**Comment by shog86** [comment]
不同的向量模型效果会有区别吗？有评测数据不

**Reply by arvinxx** [reply]
区别很大。openai 的向量化模型效果是一骑绝尘的，全语种的检索效果都很好。

而像你们之前呼声很高的那些免费或超廉价模型，往往要么只有英文效果好，要么只有中文效果好，远远不如 OpenAI 六边形。这才是我迟迟才合自定义向量化模型 PR 的根本原因。

**Reply by XiaozhongFang** [reply]
本地用vllm部署了bge-m3 embedding模型，vllm启动了embedding服务，设置了api和key，lobechat怎么只有一个环境变量指定embedding模型？在哪里设置使用自定义的embedding模型？
